snap/snapcraft.yaml

```
@@ -0,0 +1,47 @@
+name: openstack-network-agents
+summary: OVS provider-bridge helper for OpenStack Network role
+description: |
+  Provides a helper to configure a provider bridge and OVN physnet mapping.
+  This snap is meant to be driven by a subordinate charm and does not ship
+  neutron-ovn-metadata-agent.
+base: core24
+version: "0.1"
+license: Apache-2.0
+grade: devel
+confinement: strict
+
+plugs:
+  network-control:
+    interface: network-control
+  network:
+    interface: network
+  ovn-chassis:
+    interface: content
+    target: $SNAP_DATA/ovn
```

```suggestion
    target: $SNAP_DATA/microovn/chassis
```

Can you rename this path? rationale is, we might connect more interfaces from microovn snap, so let's put something consistent for the future

(I updated the previous comment, but you pushed in the meanwhile :) )

---

Yeah, now I see that :)
Updated the target and retested it as well, it works fine